policy/policy.sample.json

```diff
@@ -0,0 +1,70 @@
+{
+    "SecurityPolicies": [
+        {
+            "Name": "policy1", # names can be same for multiple tenants thus use uuid.
+            "PolicyID": "3710940D-FBC0-4A44-A95D-E549E628B4DF", # unique identifier across tenants.
+            "Associations": [ # can attach multiple tenants associations in case you want blanket policy support.
+                {
+                    "Tenant":"tenant1", #optional if TenantID below is provided 
+                    "TenantID": "3A6E467F-370B-43EA-BCEC-C28912EDD752", # optional but good to have to relate things as names are not unique
+                    "Segment": "Segment1", # optional
+                    "SegmentID": "01CA06A0-429B-4EE1-9185-F17D5CE4D0F2",
+                },
+                {
+                    "Tenant":"tenant1",
+                    "Segment": "Segment2",
+                },
+            ],
+            "Tags": [
+                {
+                    "Key": "Platform",
+                    "Value": "Openstack",
+                },
+                {
+                    "Key": "TenantID",
+                    "Value": "D46FD18B-A381-4297-A6D4-9DA594EE71C8",
+                },
+            ], # meta data attached to policies for various external enviorments like openstack/kybernetes
+            "Direction" : "Egress", # can be Egress or Ingress.
+            "CidrBlock": "0.0.0.0/0", # IP from L3 header
+            "Rules": [
+                {
+                    "Protocol": "ICMP",
+                    "FromPort": 4000,
+                    "ToPort": 4001,
```

Comment:
yes it not difficult for us as humans, but when we allow automation, these simple errors lead to deep buried failures with long stack traces, imagine bring up 100 tenants, applying these policies, and setting up the environment, the json parsing error maybe deep enough that it may fail but not bubble up to the ansible level where you actually do your machine bringup. this is just example but  the point being any and every single cause of failure will cause deployment times to be longer. parsing json error is going to be caught while applying policies while creating tenants, this is way deeper into system where a roll back or recovering from error may not be possible and you may have to restart the script after fixing this policy file.